lib/pbench/server/api/resources/upload_api.py

```diff
@@ -0,0 +1,192 @@
+import os
+import humanize
+import tempfile
+import hashlib
+from pathlib import Path
+from flask_restful import Resource, abort
+from flask import request, jsonify
+from werkzeug.utils import secure_filename
+
+ALLOWED_EXTENSIONS = {"xz"}
+
+
+class HostInfo(Resource):
+    def __init__(self, server_config, logger, api):
+        self.server_config = server_config
+        self.api = api
+        self.logger = logger
+
+    def get(self):
+        try:
+            response = jsonify(
+                dict(
+                    message=f"{self.server_config.get('user')}@{self.server_config.get('host')}"
+                    f":{self.server_config.get('pbench-receive-dir-prefix')}-002"
```

Comment:
Instead of fetching this each time, and getting the configuration error each time, perhaps we could fetch these values one in the `HostInfo` constructor, and raising the exception there if they are missing, and then just reference the constructed response.